app/client/components/productSwitch/CancellationSwitchConfirmed.tsx

```diff
@@ -55,226 +56,224 @@ const CancellationSwitchConfirmed = () => {
 	const newProduct = productSwitchConfirmationInfo.newProduct;
 
 	return (
-		<>
-			<h2 css={headingCss}>Your {newProduct.name} is now active</h2>
-			<Stack space={9}>
-				<p css={standfirstCss}>
+		<Stack space={9} cssOverrides={pageTopCss}>
```

Comment:
This additional CSS is needed to prevent the content sitting flush with the main page header. Ideally the spacing around the content would be defined as part of the overall page layout (like how we add consistent spacing to the bottom of pages), but this would require changes to a number of pages so is outside the scope of this PR.